src/wmtk/TriMeshOperation.cpp

```diff
@@ -13,22 +13,20 @@ TriMesh::TriMeshOperationState::PerFaceData TriMesh::TriMeshOperationState::get_
     Tuple t2_edge = m_mesh.switch_tuple(t, PV);
     t2_edge = m_mesh.switch_tuple(t2_edge, PE);
 
-    face_data.ears[0] =
-        EarGlobalIDs{/*.fid = */ ff_accessor.vector_attribute(t1_edge)(t1_edge.m_local_eid),
-                     /*.eid = */ m_mesh.id(t1_edge, PE)};
+    face_data.ears[0] = EarGlobalIDs{
+        /*.fid = */ ff_accessor.vector_attribute(t1_edge)(t1_edge.m_local_eid),
+        /*.eid = */ m_mesh.id(t1_edge, PE)};
 
-    face_data.ears[1] =
-        EarGlobalIDs{/*.fid = */ ff_accessor.vector_attribute(t2_edge)(t2_edge.m_local_eid),
-                     /*.eid = */ m_mesh.id(t2_edge, PE)};
+    face_data.ears[1] = EarGlobalIDs{
+        /*.fid = */ ff_accessor.vector_attribute(t2_edge)(t2_edge.m_local_eid),
+        /*.eid = */ m_mesh.id(t2_edge, PE)};
 
     return face_data;
 }
 
 // constructor
 TriMesh::TriMeshOperationState::TriMeshOperationState(TriMesh& m)
-    : flag_accessors({{m.get_flag_accessor(PrimitiveType::Vertex),
-                       m.get_flag_accessor(PrimitiveType::Edge),
-                       m.get_flag_accessor(PrimitiveType::Face)}})
+    : flag_accessors{m.get_flag_accessor(PrimitiveType::Vertex), m.get_flag_accessor(PrimitiveType::Edge), m.get_flag_accessor(PrimitiveType::Face)}
```

Comment:
@daniel-zint you were asking if perhaps your formatter is doing something different from eveyrone else's - this is a case where my formatter definitely preferred the previous code. lets double check the clang-format and whatnot soon

Reply:
We should double check that, yes, but in this specific case, I needed to change the parentheses/braces. Otherwise my compiler complained that a deleted function was called. I changed `({{` to `{`.